modules/apps/map/map-openstreetmap/src/main/resources/META-INF/resources/js/MapOpenStreetMap.es.js

```diff
@@ -41,6 +40,7 @@ class MapOpenStreetMap extends MapBase {
 		super(...args);
 
 		this._map = null;
+		this.tileURI = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
```

Comment:
Technically this value can still be set via the `args`, so we should do something like

```js
constructor(args = {}) {
    const {titleURI = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'} = args;

    this.titleURI = titleURI;
}

```

Reply:
<img src="https://media1.giphy.com/media/hFROvOhBPQVRm/giphy.gif"/>

Reply:
I think `args` can be undefined here, so you probably need to update the constructor also

```js
constructor(args = {}) {
	super(args);

        const {titleURI = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'} = args;

        this.titleURI = titleURI;
}